SU-SWS/stanford_magazine
# SOE-2354: Update default image

# READY FOR REVIEW

# Summary
- This adds a hook_update_N() command to update _yet another_ default image. 😆 

# Needed By (Date)
- Sprint end

# Criticality
- Housekeeping
- Fixes broken stuff


# Steps to Test
- Switch to this branch
- Verify state of feature & fix if necessary
`drush fd stanford_magazine`
- Clear the cache:
`drush cc all`
- Verify the old image by navigating to:
`/admin/content/default-image`
- Verify the original FID:
`drush sqlq --extra="-t" "select * from variable WHERE name = 'stanford_magazine_article_image'"`
- Run update.php
- Verify the new FID:
`drush sqlq --extra="-t" "select * from variable WHERE name = 'stanford_magazine_article_image'"`
- Verify the new image by navigating to:
`/admin/content/default-image`

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2354
## Related PRs

## More Information

## Folks to notify
@kbrownell 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
